README.md

```diff
@@ -1,2 +1,92 @@
-# ibm-ingress-nginx-operator
-operator for ingress nginx
+# IBM Ingress nginx Operator
+
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall). If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+
+The IBM Ingress nginx Repo operator provides a Helm chart repository for storing and supplying IBM and local charts.
```

Comment:
Minor edit) Using the same operator name ( Ingress Nginx) 

The IBM Ingress Nginx operator provides a Helm chart repository for storing and supplying IBM and local charts.

Reply:
Updated.